bin/hdfcoinc/pycbc_foreground_censor

```diff
@@ -30,48 +28,22 @@ logging.info('start')
 
 f = h5py.File(args.foreground_triggers, 'r')
 
-# Check if using input files from old (2-ifo) coinc code
-if 'segments/foreground_veto/start' in f:
-    start = f['segments/foreground_veto/start'][:]
-    end = f['segments/foreground_veto/end'][:]
-    vsegs = pycbc.events.start_end_to_segments(start, end)
+start = f['segments/foreground_veto/start'][:]
+end = f['segments/foreground_veto/end'][:]
+vsegs = pycbc.events.start_end_to_segments(start, end)
 
+if 'detector_1' in f.attrs:
```

Comment:
can you reinstate the comment about `# Check if using input files from old (2-ifo) coinc code`